### PR TITLE
[UPDATE] Remove deprecated code from hydrography sample

### PR DESCRIPTION
**Pull request by D-R-Smith**
### Description

- Remove deprecated code from hydrography sample.
- Use property to control progress indicator visibility
- Modularize the sample
- Changed when the progress indicator is added to the `StackPane` (I found it was not being seen because it was under the map)

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by D-R-Smith** [comment]
Assigning to @jenmerritt but really superseded by other work.

File: hydrography/add-enc-exchange-set/src/main/java/com/esri/samples/add_enc_exchange_set/AddEncExchangeSetSample.java [hunk 1]
```diff
@@ -83,10 +83,10 @@ public void start(Stage stage) {
       stackPane.getChildren().add(progressIndicator);
 
       // hide progress indicator when map is done drawing
-      mapView.addDrawStatusChangedListener(e -> {
-        if (e.getDrawStatus() == DrawStatus.IN_PROGRESS) {
+      mapView.drawStatusProperty().addListener((property, oldValue, newValue) -> {
+        if (newValue == DrawStatus.IN_PROGRESS) {
```
Comment (jenmerritt) [inline hydrography/add-enc-exchange-set/src/main/java/com/esri/samples/add_enc_exchange_set/AddEncExchangeSetSample.java]: We can do:

```
// show the progress indicator when map is drawing
progressIndicator.visibleProperty().bind(mapView.drawStatusProperty().isEqualTo(DrawStatus.IN_PROGRESS));
```

**Review by jenmerritt — COMMENTED** [review]
Can use binding for the progress indicator.

This sample also needs modularizing.

**Comment by D-R-Smith** [comment]
@jenmerritt modularized this - ready for another review.

**Review by jenmerritt — APPROVED** [review]
LGTM

**Review by Luiy0 — APPROVED** [review]
LGTM

**Pull request merged**